Analysis/Core/include/Analysis/HFConfigurables.h

```diff
@@ -0,0 +1,186 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
Hi @nzardosh, to make sure I understand the goal of this HFConfigurables.h. This .h will contain all the default parameters of the preselection cuts, which were previously set directly inside the task. They will be overwritten if a proper JSON file is given to the task. Do I understand it right?

Reply:
yes indeed this is correct